kube/patch.sh

```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
       - name: sidecar
-        image: gcr.io/stackdriver-prometheus/stackdriver-prometheus-sidecar:${SIDECAR_IMAGE_TAG}
```

Comment:
This script is used in the public docs. We shouldn't change the API. Can you preserve the current value as a default value?

Reply:
Sure, moved it over to patch.sh.